build.sbt

```diff
@@ -15,6 +15,12 @@ inThisBuild(
         "John De Goes",
         "john@degoes.net",
         url("http://degoes.net")
+      ),
+      Developer(
```

Comment:
This should be removed. Your contribution will be visible in the repo contributors.